semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -237,7 +237,14 @@ def parse_id(attribute):
             and examples is not None
         ):
             examples = [examples]
-        if is_simple_type and attr_type not in ["boolean", "boolean[]"]:
+        if is_simple_type and attr_type not in [
+            "boolean",
+            "boolean[]",
+            "int",
+            "int[]",
+            "double",
+            "double[]",
+        ]:
```

Comment:
It seems to me that we should change this to

```suggestion
        if is_simple_type and attr_type in ["string", "string[]"]:
```

(not sure if we need the `is_simple_type and`).

Reply:
I am more in favor of explicitly excluding the types for which an example is not needed. 
`is_simple_type` is required because enums do not need an example.

Reply:
OK for me. But maybe add a comment like: "# Don't enforce examples for numeric and boolean".